event_attribution_reporting.md

```diff
@@ -0,0 +1,136 @@
+
+
+## Browsing history reconstruction
+
+Reporting attribution without a pre-existing navigation allows the `reportingorigin` to learn whether a given user on the source site visited the attributiondestination site at all. In the Click through API, this is not an issue because the `reportingorigin` knows a priori the user was navigating to attributiondestination.
+
+This new threat is/can be mitigated in a number of ways:
+
+### Plausible Deniability
+
+By adding noise to whether an attribute source gets [triggered](#noise-on-the-attribution-triggering-event), a `reportingorigin` will not know with absolute certainty whether a user actually visited a site. In fact, this additional mitigation gives some local differential privacy guarantees on the data in the API.
+
+It becomes important for browsers to ensure that other API parameters do not allow a `reportingorigin` to easily denoise reports for a given `attributionsourceeventid`. For example, if multiple reports were sent for each source it may allow reporters to use the number of reports as a mechanism to remove noise, and may require browsers to implement more complex noise logic.
+
+Note: Since conversion rates in practice are extremely rare events, we need to be careful to set the noise parameter so as not to ruin the utility of this API (i.e. noise drowns out signal).
+
+### Limiting the number of unique sites covered by pending sources
+
+To limit the breadth of attributiondestination sites that a `reportingorigin` may be trying to measure user visits on, the browser can limit the number attributiondestination eTLD+1s represented by pending sources for a publisher.
+
+The browser can place a limit on the number of a source site's pending source's unique attributiondestinations. When an attribution source is registered for an eTLD+1 that is not already in the pending sources and a source site is at its limit, the browser will delete the oldest pending attribution source from the source site. If after deleting the oldest source, the source site is still at its limit, the browser will continue to delete pending sources until it is able to make room for the newly registered source.
+
+The lower this value, the harder it is for a `reportingorigin` to use the API to try and measure user browsing activity not associated with ads being shown. Browsers may choose parameters on the number of attributiondestinations to make their own tradeoffs for privacy and utility.
+
+Because this limit is per source site, it is possible for different reportingorigins on a site to push the other attribution sources out of the browser. To limit abuse by potentially bad actors, the browser could consider adding a publisher side configuration which could partition this queue into smaller queues managed per reportingorigin.
```

Comment:
```suggestion
Because this limit is per source site, it is possible for different `reportingorigin`s on a site to push the other attribution sources out of the browser. To limit abuse by potentially bad actors, the browser could consider adding a source-site side configuration which could partition this queue into smaller queues managed per `reportingorigin`.
```

Reply:
Was thinking possibly using 'source-site' instead of publisher but I don't have a strong preference either way.

Reply:
source-site seems better. Adjusted above as well.